src/component/mxgraph/shape/event-shapes.ts

```diff
@@ -125,8 +122,8 @@ export class EventShape extends mxgraph.mxEllipse {
  * @internal
  */
 export class EndEventShape extends EventShape {
-  constructor() {
-    super();
+  constructor(bounds: mxRectangle, fill: string, stroke: string, strokewidth?: number) {
```

Comment:
**thought:** personal quote, I don't understand why this changes. Need to test locally to see the eslint error in action. This makes things harder to read.
The idea was to have a constructor without parameter for simplicity. In practice, mxGraph call the constructor without parameters. The remaining parameters are probably the rest of an old usage of Shapes, which are now always instantiated by the CellRenderer.

Reply:
![](https://github.com/process-analytics/bpmn-visualization-js/assets/4921914/3bae7eb9-e7bc-4b07-8b62-4ab985a137c4)

Reply:
It's because we want to override the value of the property `withFilledIcon`in the constructor.  
Maybe, we should to set it in another way.

Reply:
It's because we want to override the value of the property `withFilledIcon`in the constructor.  
Maybe, we should to set it in another way.